### PR TITLE
Add message for duplicate of MC-108

### PR DESCRIPTION
**Pull request by chandler05**
WAI duplicate message with a link to the quasi-connectivity wiki article

### CODE REVIEW

File: assets/messages.yml [hunk 1]
```diff
@@ -351,6 +351,20 @@ messages:
...
         Please take a look at the Moderator’s Note on that ticket to see solutions to your issue.
 
+        %quick_links%
+      fillname: []
+  duplicate-of-mc-108:
+    - project: mc
+      name: Duplicate of MC-108
+      shortcut: quasi
+      message: |-
+       *Thank you for your report!*
```
Comment (Marcono1234) [inline assets/messages.yml:361]: Looks like the indentation here is not correct.
```suggestion
        *Thank you for your report!*
```

File: assets/messages.yml [hunk 2]
```diff
@@ -351,6 +351,20 @@ messages:
...
 
+        %quick_links%
+      fillname: []
+  duplicate-of-mc-108:
+    - project: mc
+      name: Duplicate of MC-108
+      shortcut: quasi
+      message: |-
+        *Thank you for your report!*
+        We're tracking this issue as *%s%*, so this ticket is being resolved and linked as a *duplicate*.
```
Comment (SPGoding) [inline assets/messages.yml:362]:
```suggestion
        We're tracking this issue as *MC-108*, so this ticket is being resolved and linked as a *duplicate*.
```

Comment (chandler05) [reply assets/messages.yml:362]: Ah yes, that would be pretty smart to do

**Comment by NeunEinser** [comment]
Omg, this is #108 and it was about adding a message dedicated to MC-108 :O

**Comment by chandler05** [comment]
:O not planned I swear xD

File: assets/messages.yml [hunk 3]
```diff
@@ -351,6 +351,20 @@ messages:
...
+    - project: mc
+      name: Duplicate of MC-108
+      shortcut: quasi
+      message: |-
+        *Thank you for your report!*
+        We're tracking this issue as *MC-108*, so this ticket is being resolved and linked as a *duplicate*.
+
+        That ticket has already been resolved as working as intended, which means this is not considered a bug and won't be fixed. Please do not leave a comment on the linked ticket.
+
+        If you would like to learn more about this particular feature, see [this article|https://minecraft.gamepedia.com/Tutorials/Quasi-connectivity] on the Minecraft Wiki.
```
Comment (NeunEinser) [inline assets/messages.yml:366]: I would substitute "behavior" for "feature" here.
For people who don't expect this, calling it a feature is a bit far fetched, and it's more of a bug that turned out useful and was not removed.

Also I get tech-community-vibes from this. Certain members of that community like to complain about certain """""features""""" being removed.

Comment (Marcono1234) [reply assets/messages.yml:366]: Would you mind creating a pull request for it?

**Pull request merged**